bin/index.js

```diff
@@ -40,7 +42,16 @@ if (argv.dest) {
 } else {
   var topLevel = exec('git rev-parse --show-toplevel', { silent: true })
     .output.slice(0, -1);
-  dest = topLevel + '/.git/hooks/';
+  if(test('-f', topLevel + '/.git')){
+    // this is a sub module
+    var buf = fs.readFileSync(topLevel + '/.git', "utf8").trim();
+    if(buf.substr(0,6) == 'gitdir'){
```

Comment:
Please follow existing style (spaces after keywords and before opening curly), and use `===` only.